app/views/BookDetail/index.tsx

```diff
@@ -107,15 +108,18 @@ function BookDetail() {
                                 )}
                                 footerIcons={(
                                     <>
-                                        <Button name="buy">
+                                        <ButtonLikeLink
+                                            variant="primary"
+                                            to={`/order?book=${result.book.id}`}
+                                        >
                                             Buy now
-                                        </Button>
-                                        <Button
-                                            name="wishlist"
-                                            variant="secondary"
+                                        </ButtonLikeLink>
+                                        <ButtonLikeLink
+                                            variant="primary"
+                                            to="/wishlist-page"
```

Comment:
Let's not prepare link directly. Please use path from routes page

Reply:
this is fixed in PR #16